.github/actions/ensure-stable-pr/action.yml

```
@@ -0,0 +1,21 @@
+name: Ensure Stable PR
+description: Ensure that the PR hasn't changed since the commented command was initiated
+inputs:
+  cmd:
+    description: 'The command used in the comment'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Ensure the PR hasn't changed since initiating the commented command.
+      uses: actions/github-script@v7
+      env:
+        CMD: ${{ inputs.cmd }}
+      with:
+        script: |
+          const prLastUpdated = new Date(github.pull_request.updated_at);
```

```suggestion
          const prLastUpdated = new Date(github.pull_request.pushed_at);
```

If we're only interested in commits, we can use `pushed_at`. Might reduce false-positives